index.js

```diff
@@ -11,7 +11,8 @@ const DEVICE_TYPES = {
   'SIREN1': 'Siren',
   'SMARTPLUG': 'Smart plug',
   'SMOKE2': 'Rökdetektor',
-  'VOICEBOX1': 'Directenhet'
+  'VOICEBOX1': 'Directenhet',
+  'HUMIDITY1': 'Luftfuktighet'
```

Comment:
Can we just move this to the top of the list? Trying to keeping them sorted 👍 

If this device is a [Klimatdetektor](https://www.verisure.se/smarta-hem/klimat.html), I think we should call it that. Instead of "Luftfuktighet".